### ISSUE TITLE
The bpf framework requires 'bpftool gen'

### ISSUE THREAD

**Issue by evverx** [systemd/systemd#22051]
For the bpf framework to work bpftool has to support the gen command. Without it `ninja` fails to build `systemd` with
```
[462/1877] /home/vagrant/systemd/tools/build-bpf-skel.py --clang_exec /usr/bin/clang --llvm_strip_exec /usr/bin/llvm-strip --bpftool_exec /usr/sbin/bpftool --arch x86_64 ../src/core/bpf/restrict_ifaces/restrict-ifaces.bpf.c src/core/bpf/restrict_ifaces/restrict-ifaces.skel.h
FAILED: src/core/bpf/restrict_ifaces/restrict-ifaces.skel.h
/home/vagrant/systemd/tools/build-bpf-skel.py --clang_exec /usr/bin/clang --llvm_strip_exec /usr/bin/llvm-strip --bpftool_exec /usr/sbin/bpftool --arch x86_64 ../src/core/bpf/restrict_ifaces/restrict-ifaces.bpf.c src/core/bpf/restrict_ifaces/restrict-ifaces.skel.h
Usage: /usr/lib/linux-tools/5.4.0-92-generic/bpftool [OPTIONS] OBJECT { COMMAND | help }
       /usr/lib/linux-tools/5.4.0-92-generic/bpftool batch file FILE
       /usr/lib/linux-tools/5.4.0-92-generic/bpftool version

       OBJECT := { prog | map | cgroup | perf | net | feature | btf }
       OPTIONS := { {-j|--json} [{-p|--pretty}] | {-f|--bpffs} |
	            {-m|--mapcompat} | {-n|--nomount} }
Traceback (most recent call last):
  File "/home/vagrant/systemd/tools/build-bpf-skel.py", line 128, in <module>
    bpf_build(args)
  File "/home/vagrant/systemd/tools/build-bpf-skel.py", line 92, in bpf_build
    gen_bpf_skeleton(bpftool_exec=args.bpftool_exec,
  File "/home/vagrant/systemd/tools/build-bpf-skel.py", line 63, in gen_bpf_skeleton
    skel = subprocess.check_output(bpftool_args, universal_newlines=True)
  File "/usr/lib/python3.8/subprocess.py", line 415, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/usr/sbin/bpftool', 'g', 's', '../src/core/bpf/restrict_ifaces/restrict-ifaces.bpf.o']' returned non-zero exit status 255.
ninja: build stopped: subcommand failed.
```

It was discovered in https://github.com/systemd/systemd/pull/22021#issuecomment-1007918469

**Comment by wat-ze-hex** [comment]
I'll take a look

**Comment by evverx** [comment]
@wat-ze-hex thanks!

On an absolutely unrelated note, `libbpf` is on OSS-Fuzz now and I wonder if I should add your email address to https://github.com/google/oss-fuzz/blob/master/projects/libbpf/project.yaml? It was integrated kind of fast there and I completely forgot to ask who else should be on the list. It should be a gmail address due to https://google.github.io/oss-fuzz/faq/#why-do-you-require-a-google-account-for-authentication though. Anyway, just let me know if you're interested.

**Comment by wat-ze-hex** [comment]
@evverx 

Yes, please add me. The e-mail is in my profile.

**Comment by yuwata** [comment]
Fix is waiting in #22093.
